src/test/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleFinalOrderTest.java

```diff
@@ -184,4 +189,25 @@ void shouldSendNotificationsInSubmittedCallback() {
 
         verify(notificationDispatcher).send(switchedToSoleFoNotification, caseDetails.getData(), caseId);
     }
+
+    @Test
```

Comment:
You don't need two tests here, you just need one to verify that GeneralReferralService has been called 
https://github.com/hmcts/nfdiv-case-api/blob/master/src/test/java/uk/gov/hmcts/divorce/common/event/ApplyForFinalOrderTest.java#L162